### PR TITLE
Adds a pair of pink programming socks that can be found in the RD's locker

### PR DESCRIPTION
**Pull request by RFultra**
:cl:
 * rscadd: Added a pair of pink programming socks that can be found in the RD's locker

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 1]
```diff
@@ -47,3 +47,4 @@
 	new /obj/item/clothing/mask/gas(src)
 	new /obj/item/device/flash(src)
 	new /obj/item/weapon/switchtool/holo(src)
+	new /obj/item/clothing/shoes/kneesocks/pinkknee(src)
```
Comment (MadmanMartian) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: Why not just have the subtype as kneesocks/pink?

pink knee sounds like an ailment

Comment (RFultra) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: I did it so the name is the same is the icons in the dmi

Comment (MadmanMartian) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: Right, the image has to have a short and descriptive name (It's pink and has something to do with knees) but right here you've said it's something to do with knees twice.

shoes don't do this, shoes are just shoes/black, shoes/orange, shoes/brown

Comment (RFultra) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm]: there, fixed

**Comment by yclatious** [comment]
Repoban when

**Comment by Chucklemonger** [comment]
what makes them programming socks

**Comment by MadmanMartian** [comment]
That's a good point, give them a research level, programming = 2 or something

**Comment by Moonman1946** [comment]
let's not do that today

**Comment by chatsonf** [comment]
I can already see waifus maining RD, or spending a large amount of time trying to get these.

Make 'em male only so it fits the meme and we got a deal.

**Comment by DrSnips** [comment]
I'd give them at least a 4 or 5, a micro manipulator is worth 2 and you cant do much after programming 4 (mostly circuts).

**Comment by jknpj** [comment]
Even if it's a meme, you're pretty much just doing it to toe the line

**Comment by yclatious** [comment]
Water is wet, sun shines, grass grows, goofball is tistic and rf codes memes and baits a codeban.

God bless.

**Comment by SonixApache** [comment]
need to add a sailor shirt that doesn't come with pants alongside this or it's pointless

**Comment by troots** [comment]
@SonixApache is an expert in this field so i'd listen to him tbh

**Comment by DrSnips** [comment]
The RD locker has a suit and skirt fyi.

**Comment by SonixApache** [comment]
for the confused among us
shit meme

On a semi-related note, please nuke this PR post-haste.

**Comment by ValkyrieSkies** [comment]
Should've included blue and black as well, would've guaranteed a merge.

**Comment by RFultra** [comment]
Can we get confirmation on this, if I add blue and black kneesocks, will they get merged?

**Comment by Exxion** [comment]
Meme PRs are fun and all
But not if you make them constantly and never make an actual PR

**Comment by Nickvr628** [comment]
Watch me port this to yogstation.

**Pull request closed without merging**